src/cf-expandables/src/Expandable.js

```diff
@@ -12,11 +13,10 @@ const removeClass = domClassList.removeClass;
 const closest = require(
   'cf-atomic-component/src/utilities/dom-closest'
 ).closest;
-const ExpandableTransition = require(
-  'cf-atomic-component/src/utilities/transition/ExpandableTransition'
-);
+const fnBind = require( 'cf-atomic-component/src/utilities/function-bind' ).bind;
```

Comment:
This is fine for this PR, but I think we should probably deprecate the `function-bind` module since it's just needed for IE8.

Reply:
I found it was necessary in order to pass `this` to `_accordionActivatedHandler` without making it a public method following the example from ExpandableTransition

Reply:
Ah, sorry, I didn't look at what `fnBind` was doing under the hood. I'll see if I can include it here so we're not sending out another major release of atomic component